docroot/sites/now.uiowa.edu/modules/now_core/templates/views-view-unformatted--iowa-now-experts.html.twig

```diff
@@ -0,0 +1,34 @@
+{{ attach_library('uids_base/person') }}
+{% if title %}
+  <h3>{{ title }}</h3>
+{% endif %}
+{% for row in rows %}
+  {%
+    set row_classes = [
+    default_row_class ? 'views-row',
+  ]
+  %}
+
+  {% embed '@uids_base/uids/card.html.twig' with row.card_options %}
+
+    {% block card_img %}
+    {% if card_image|striptags('<img><picture>')|trim is not empty %}
+      {{ parent() }}
+    {% else %}
+      <div {{ media_attributes.addClass(media_styles, media_sizes, 'card__media') }}>
+        <div class="img--empty">&nbsp;</div>
+      </div>
+    {% endif %}
+    {% endblock %}
+
+    {% block card_meta %}
+      {% if person_email|striptags|trim is not empty %}
+        {{ person_email }}
+      {% endif %}
+      {% if person_phone|striptags|trim is not empty %}
+        {{ person_phone }}
+      {% endif %}
+    {% endblock %}
```

Comment:
@bspeare I'm wondering if you are doing this to avoid the `<div class="card__details"><div class="card__meta"></div></div>` markup around these elements? Otherwise, see my suggestion for `now_core.module`.